server/views/referrals/show/offenceHistory.njk

```diff
@@ -0,0 +1,50 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
```

Comment:
Might be good to squash this commit into the next one (that adds the controller action) so it's a bit clearer to see what the functions in this file are doing.